tests/api/controllers/test_run.py

```diff
@@ -0,0 +1,59 @@
+import json
+import time
+
+from tests.api.controllers import APITest
+
+
+class TestRun(APITest):
+
+    def _wait_for_run_complete(self, id):
+        counter = 0
+        while 1:
+            res = self.app.get('/runs/{}'.format(id))
+            res_dict = json.loads(res.data)
+            if res_dict['state'] != 'RUNNING':
+                break
+            time.sleep(self.wait_interval)
+            if counter < self.counter:
+                continue
+        self.assertNotEqual('RUNNING', res_dict['state'])
+
+    def test_get_empty_runlist(self):
+        response = self.app.get('/runs')
+        self.assertEqual(200, response.status_code)
+        self.assertEqual('[]\n', response.data)
+
+    def test_get_run_list(self):
+        pb = 'tests/fixtures/playbooks/hello_world.yml'
+        response = self.app.post('/runs', data=json.dumps(
+                                            dict(playbook_path=pb,
+                                            inventory_file='localhosti,',
+                                            connection='local')),
+                                            content_type='application/json')
+        res_dict = json.loads(response.data)
+        run_id = res_dict['id']
+        self.assertEqual(200, response.status_code)
+        response = self.app.get('/runs')
+        res_list = json.loads(response.data)
+        found = False
+        for item in res_list:
+            if item['id'] == run_id:
+                found = True
+                break
+        self.assertEqual(True, found)
+        self._wait_for_run_complete(run_id)
+
+    def test_trigger_run(self):
+        pb = 'tests/fixtures/playbooks/hello_world.yml'
+        response = self.app.post('/runs', data=json.dumps(
+                                            dict(playbook_path=pb,
+                                            inventory_file='localhosti,',
+                                            connection='local')),
+                                            content_type='application/json')
+        res_dict = json.loads(response.data)
+        self.assertEqual(200, response.status_code)
```

Comment:
Instead of 200, just use constants from http_client

from six.moves import http_client

http_client.OK = 200